package.json

```diff
@@ -333,5 +335,6 @@
   "engines": {
     "node": "20.14.0",
     "npm": "10.7.0"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
This change looks like it snuck in by mistake - can we please remove it?

Reply:
ah, thanks for catching that, removed it